PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp

```diff
@@ -848,8 +849,62 @@ INT_PTR CALLBACK FindReplaceDlg::run_dlgProc(UINT message, WPARAM wParam, LPARAM
 			return TRUE;
 		}
 
+		case WM_CTLCOLOREDIT:

```

Comment:
Isn't it the same code as in your other PR? 
https://github.com/notepad-plus-plus/notepad-plus-plus/pull/9800/files#diff-62585f441298d642a3dbd871af2800eac3606f9fe148372ab119feed21e050b0

Reply:
Yeah, there are a bunch of conflicts if I rebase on the current master, once #9800 is merged I'll rebase this one and mark it as non-draft.